PowerEditor/src/Notepad_plus.cpp

```diff
@@ -2452,123 +2452,38 @@ void Notepad_plus::setUniModeText()
 }
 
 
-void Notepad_plus::addHotSpot()
+void Notepad_plus::addHotSpot(ScintillaEditView* view)
 {
-	int startPos = 0;
-	int endPos = -1;
-	auto endStyle = _pEditView->execute(SCI_GETENDSTYLED);
-
-	_pEditView->getVisibleStartAndEndPosition(&startPos, &endPos);
-
-	_pEditView->execute(SCI_SETSEARCHFLAGS, SCFIND_REGEXP|SCFIND_POSIX);
-
-	_pEditView->execute(SCI_SETTARGETRANGE, startPos, endPos);
-
-	std::vector<unsigned char> hotspotPairs; //= _pEditView->GetHotspotPairs();
-
-	unsigned char style_hotspot = 0;
-	unsigned char mask = 0x40; // INDIC1_MASK;
-	// INDIC2_MASK == 255 and it represents MSB bit		
-	// only LEX_HTML and LEX_POSTSCRIPT use use INDIC2_MASK bit internally		
-	// LEX_HTML is using INDIC2_MASK bit even though it has only 127 states, so it is safe to overwrite 8th bit		
-	// INDIC2_MASK will be used for LEX_HTML		
-
-	// LEX_POSTSCRIPT is using INDIC2_MASK bit for "tokenization", and is using mask=31 in lexer,		
-	// therefore hotspot in LEX_POSTSCRIPT will be saved to 5th bit		
-	// there are only 15 states in LEX_POSTSCRIPT, so it is safe to overwrite 5th bit		
+	ScintillaEditView* _pView = view ? view : _pEditView;

```

Comment:
please use `pView` instead of "_pView".
_varNames is the member of class in the convention in Notepad++ project.